.vortex/cli/src/Form/VortexForm.php

```
@@ -0,0 +1,267 @@
+   * @throws \RuntimeException
+   *   When a question id has no handler behind it.
+   */
+  public static function handlers(Config $config): array {
+    $registry = (new Tui(self::create($config), [self::HANDLER_NAMESPACE]))->registry();
+    $handlers = [];
+
+    foreach (array_keys(self::WEIGHTS) as $id) {
+      $class = $registry->resolve($id);
+
+      if ($class === NULL || !is_a($class, HandlerInterface::class, TRUE)) {
+        throw new \RuntimeException(sprintf('Handler for "%s" not found.', $id));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '165', column '19'). (undefined)

(MissingImport)